### DISCUSSION TITLE
Add support for some emacs-like keybindings

### DISCUSSION THREAD

**Discussion by mrzool** [monkeytypegame/monkeytype#1903] [Ideas] [answered]
When typing on my Mac I never use backspace to delete back one character or the arrow keys to move around. I prefer to use the standard emacs-like keybindings available in every native macOS application.

That means, for example, that I use `<C-h>` instead of backspace to delete back one character, `<C-w>` to delete back one word, `<C-a>` and `<C-e>` to jump to the beginning and to the end of the line respectively, or `<C-b>` and `<C-f>` to move my cursor back and forward.  This allows me to keep my fingers on the home row pretty much all the time, and it makes moving and editing around much more efficient and fun (of course, my caps lock key is remapped to work as control key).

I think it would be great if monkeytype supported at least a subset of these keybindings (`<C-h>` to delete back one character being the most obvious candidate).

It required a fair amount of training to stop using the backspace key and start using `<C-h>` instead (which involved deactivating my backspace key system-wide for a while), and now that I started practicing on monkeytype I found myself slowly slipping back into my old habit, which is unwelcome news.

**Comment by SeerLite** [comment, marked as answer]
If I understand correctly, `<C-h>` should work in regular browser inputs too, right? Then does it work like you expect on the preview page of #1325? You can test it [here](https://seerlite.github.io/monkeytype/issue/531-input-rewrite/).

**Reply by mrzool** [reply]
Yes! Every keybinding seems to work out of the box on that test page. I'll be very happy when that PR gets merged :)
